src/generator/protocol/operations.ts

```diff
@@ -140,7 +148,15 @@ function formatParamValue(param: Parameter, imports: ImportManager): string {
       // cannot use formatConstantValue() since all values are treated as strings
       return `"${constSchema.value.value}"`;
     case SchemaType.Date:
+      return `${paramName}.Format("${dateFormat}")`;
     case SchemaType.DateTime:
+      if (paramName[0] == '*') {
+        paramName = paramName.substr(1);
+      }
+      if ((<HeaderFormat>param.schema).format === 'date-time-rfc1123') {
```

Comment:
Minor issue, there should be an existing definition for `date-time-rfc1123` in autorest.core someplace.

Reply:
I did try to find it in the schema types before, but I'll take a look elsewhere for it then.

Reply:
So this is what I've found so far in reference to the alternate datetime format: 
```
/** a schema that represents a DateTime value */
export interface DateTimeSchema extends PrimitiveSchema {
    /** the schema type  */
    type: SchemaType.DateTime;
    /** date-time format  */
    format: 'date-time-rfc1123' | 'date-time';
}
```

Reply:
Doesnt seem like it has its own specific interface, I also did a search in the node_modules folder to find any other reference to it and it seems that that is the only one.